src/android/HockeyApp.java

```diff
@@ -115,6 +172,55 @@ public void run() {
             return true;
         }
         
+        if (action.equals("composeFeedback")) {
+            final ArrayList<Uri> attachments = new ArrayList<Uri>();
+            final Activity context = cordova.getActivity();
+            boolean attachScreenshot = args.optBoolean(0);
+            if (attachScreenshot) {
+                Bitmap screenshot = getBitmap();
+                if (screenshot == null) {
+                    callbackContext.error("failed to take screenshot");
+                    return false;
+                }
+                try{
+                    File imageFile = File.createTempFile("screen", ".jpg", context.getFilesDir());
```

Comment:
Should we name this something like `HockeySDK-screen.jpg`, just to guard against the unlikely event that some other process is using the same name?